test/e2e/configurer/chain/queries.go

```diff
@@ -413,3 +415,54 @@ func (n *NodeConfig) QueryWasmSmartObject(contract string, msg string) (resultOb
 	}
 	return resultObject, nil
 }
+
+func (n *NodeConfig) QueryProposal(proposalNumber int) govtypesv1.QueryProposalResponse {
```

Comment:
I wonder whether all of those commands are necessary, from what I can tell they are not used in any way ?

Reply:
nop, they are mostly not needed 
just changes were applied when cherry picked, I will remove unused funcs

Reply:
done at https://github.com/babylonlabs-io/babylon/pull/46/commits/4bf9af900da69442dcb80d365c8e43b064ce0b3a